model/parVarSheet.cc

```diff
@@ -72,6 +74,38 @@ namespace minsky
   
     return background;
   }
+  
+namespace
+{
+  std::string definition(const string& valueId)
+  {
+    SystemOfEquations system(cminsky());	  
+	ostringstream o;
+
+    for (const VariableDAG* i: system.variables)
+      {
+		if (i->valueId==valueId) 
+		{  		  
+           if (dynamic_cast<const IntegralInputVariableDAG*>(i) ||
+               !i || i->type==VariableType::constant) continue;
+           if (i->rhs)
+             o << i->rhs->matlab();
+	    }
+      }
+
+    for (const VariableDAG* i: system.integrationVariables)
+      {
+		if (i->valueId==valueId) 
+		{  
+            VariableDAGPtr input=system.expressionCache.getIntegralInput(i->valueId);
+            if (input && input->rhs)
+              input->rhs->matlab(o);
+	    }
+      }
```

Comment:
This does work, but I'm sure you appreciate won't scale well for large models. There's two features you could use to look up the nodes defining a variable in O(log(N)), rather than O(N). The first is SystemOfEquations::getNodeFromWire, which requires a wire to look up. The other is to use the subexression cache directly: You can look up a variable's definition by name - you might need to know whether you're starting with a stock or a flow variable first.

Currently, permissions won't allow you to do this from parVarSheet, so you'll need to rearrange things a bit. Maybe add another method to SystemOfEquations?